lord/time_lord_test.go

```diff
@@ -187,6 +196,48 @@ var _ = DescribeTable("A range without a previous time", (testCase).Run,
 	}),
 )
 
+var _ = DescribeTable("A range with a previous time", (testCase).Run,
+	Entry("an hour before start", testCase{
+		start:  "2:00 AM +0000",
+		stop:   "4:00 AM +0000",
+		now:    "3:00 AM +0000",
+		prev:   "1:00 AM +0000",
+		result: true,
+	}),
+	Entry("with stop before start and prev in the start day and now in the stop day", testCase{
+		start:  "10:00 AM +0000",
+		stop:   "5:00 AM +0000",
+		now:    "4:00 AM +0000",
+		nowDay: time.Tuesday,
+		prev:   "11:00 AM +0000",
+		prevDay: time.Monday,
+		result: false,
```

Comment:
Just writing my thoughts since this is pretty confusing (time is hard).

This is correct because 10AM - 5AM actually means the following range: 10A, **11A**, 12P, 1P, 2P, 3P, 4P, 5P, 6P, 7P, 8P, 9P, 10P, 11P, 12A, 1A, 2A, 3A, **4A**, 5A.

If the previous time is 11AM on Monday, and the resource returned a new version at 4AM on Tuesday that'd be "double counting" for one time range (the bolded entries).